tests/integration/test_heartbeats.py

```diff
@@ -19,8 +20,6 @@
 if os.path.isfile("/etc/conda/machine_token"):
     expected += ("m",)
 
-os.environ["ANACONDA_ANON_USAGE_DEBUG"] = "1"
-os.environ["ANACONDA_HEARTBEAT_DRY_RUN"] = "1"
```

Comment:
No need for these anymore, because we are mining the heartbeat information directly from the requests actually made.